static/styles/subscriptions.css

```diff
@@ -1019,12 +1015,12 @@
 }
 
 #subscription_overlay .stream-types,
-#subscription_overlay ul {
+#subscription_overlay .subsection-parent {
     margin: 0;
     padding: 5px 10px;
 }
 
-#subscription_overlay ul > li {
+#subscription_overlay .subsection-parent > div {
```

Comment:
We never want CSS like this, that refers to a specific object just be it being a div inside some other object, because that's hard to read and does unpredictable things if we later e.g. add a tooltip or modal inside this parent.  I think you want `.input-group` instead of `> div` here, and then possibly adding the `input-group` class to the new div for the streamcolor input.